File: PRAKTIKUM MODUL 6/PRAK601_2310817220028_SheilaSabina.py
def create_matrix(rows, cols, nilai):
    matriks = []
    indeks = 0

    for i in range(rows):
        row = []
        for j in range(cols):
            row.append(nilai[indeks])
            indeks += 1
        matriks.append(row)

    return matriks

def print_matrix(matriks):
    for row in matriks:
        print(" ".join(map(str, row)))

print("Input")
rows, cols = map(int, input().split())
nilaimatriks = list(map(int, input().split()))

matriks = create_matrix(rows, cols, nilaimatriks)

print("\nOutput")
print_matrix(matriks)

File: PRAKTIKUM MODUL 6/PRAK601_2310817220028_SheilaSabina.c
#include <stdio.h>

void createMatrix(int rows, int cols, int nilai[], int matriks[rows][cols]) {
    int indeks = 0;

    for (int i = 0; i < rows; i++) {
        for (int j = 0; j < cols; j++) {
            matriks[i][j] = nilai[indeks];
            indeks++;
        }
    }
}

void printMatrix(int rows, int cols, int matriks[rows][cols]) {
    for (int i = 0; i < rows; i++) {
        for (int j = 0; j < cols; j++) {
            printf("%d ", matriks[i][j]);
        }
        printf("\n");
    }
}

int main() {
    int rows, cols;

    printf("Input\n");
    scanf("%d %d", &rows, &cols);

    int nilaimatriks[rows * cols];

    for (int i = 0; i < rows * cols; i++) {
        scanf("%d", &nilaimatriks[i]);
    }

    int matriks[rows][cols];
    createMatrix(rows, cols, nilaimatriks, matriks);

    printf("\nOutput\n");
    printMatrix(rows, cols, matriks);

    return 0;
}